kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/datadictionary/PurchaseOrderDocument.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xmlns:dd="http://rice.kuali.org/dd" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd         http://rice.kuali.org/dd         http://rice.kuali.org/dd/dd.xsd">
+
+  <bean id="PurchaseOrderDocument" parent="PurchaseOrderDocument-parentBean">
+    <property name="documentClass" value="edu.arizona.kfs.module.purap.document.PurchaseOrderDocument"/>
+    <property name="baseDocumentClass" value="edu.arizona.kfs.module.purap.document.PurchaseOrderDocument"/>
+    <property name="attributes">
+      <list merge="true">
+        <ref bean="PurchaseOrderDocument-routeCode"/>
+      </list>
+    </property>
+  </bean>
+
+<!-- Attribute Definitions -->
+   <bean id="PurchaseOrderDocument-routeCode" parent="RequisitionDocument-routeCode-parentBean"/>
+   
+   <bean id="RequisitionDocument-routeCode-parentBean" abstract="true" parent="AttributeDefinition"> 
```

Comment:
Should be:
bean id="PurchaseOrderDocument-routeCode" parent="PurchaseOrderDocument-routeCode-parentBean"
bean id="PurchaseOrderDocument-routeCode-parentBean" abstract="true" parent="AttributeDefinition">